src/ORM/Association/SelectableAssociationTrait.php

```diff
@@ -82,8 +82,11 @@ protected function _buildQuery($options)
 
         $finder = isset($options['finder']) ? $options['finder'] : $this->finder();
         list($finder, $opts) = $this->_extractFinder($finder);
+        $options += ['fields' => []];
+
         $fetchQuery = $this
             ->find($finder, $opts)
+            ->select($options['fields'])
```

Comment:
No changes to previous behavior. That call was done before later in that method. I just added it here for clarity